src/tmp.jl

```diff
@@ -240,26 +240,52 @@ function tmp_create_union_tables(connection)
     # These are equivalent to the partitions in time-resolution.jl
     # But computed in a more general context to be used by variables as well
 
+    # Incoming flows
+    DuckDB.execute(
+        connection,
+        "CREATE OR REPLACE TEMP TABLE t_union_in_flows AS
```

Comment:
Is it okay to just use `CREATE`?

Reply:
It should be, in the final version we can change to that, if preferred. However, for development it makes it easier to rerun some functions without rerunning everything. If only `CREATE` is used and the table exists, it will fail and then you have to manually delete the table or create a new `connection` object.